app/src/main/java/de/danoeh/antennapod/preferences/PreferenceUpgrader.java

```diff
@@ -117,5 +120,15 @@ private static void upgrade(int oldVersion, Context context) {
                 prefs.edit().putString(UserPreferences.PREF_DRAWER_FEED_COUNTER, "2").apply();
             }
         }
+        if (oldVersion < 2070095) {
+
+            long value = Long.parseLong(SleepTimerPreferences.lastTimerValue());
+            TimeUnit unit = SleepTimerPreferences.UNITS[SleepTimerPreferences.lastTimerTimeUnit()];
```

Comment:
This is now the only usage of `SleepTimerPreferences.lastTimerTimeUnit()`. Please instead move the code here and remove it from the class, so that the "main code" does not have unused legacy code. All legacy code should be bundled here in the upgrader.